helm-charts/common/vllm/README.md

```diff
@@ -10,23 +10,43 @@ To install the chart, run the following:
 
 Note that you cannot use vllm as the service release name due to [environment variables conflict](https://docs.vllm.ai/en/stable/serving/env_vars.html#environment-variables).
 
-```console
+```bash
 cd GenAIInfra/helm-charts/common
 export MODELDIR=/mnt/opea-models
 export MODELNAME="Intel/neural-chat-7b-v3-3"
 export HFTOKEN="insert-your-huggingface-token-here"
+
+# If you are behind a proxy, please export the appropriate proxy values.
+export http_proxy=<your_http_proxy>
+export https_proxy=<your_https_proxy>
+
```

Comment:
IMHO general deployment info should be in general doc, not spread to all components.  Add a link to common doc if needed.